requirements/requirements.txt

```diff
@@ -26,7 +26,6 @@ gunicorn==18.0
 lxml==3.4.4
 kafka-python==0.9.4
 mock==0.8.0
```

Comment:
:+1: not my area of expertise but agree. cc @nickpell @proteusvacuum @benrudolph who i think are the ones who have most recently been in the billing tests